core/meta/src/TDataMember.cxx

```diff
@@ -887,7 +887,7 @@ TMethodCall *TDataMember::SetterMethod(TClass *cl)
          settername.Form( "Set%s", dataname+1);
          if (strstr(settername, "Is")) settername.Form( "Set%s", dataname+3);
          if (GetClass()->GetMethod(settername, "1"))
```

Comment:
```suggestion
         if (GetClass()->GetMethod(settername, "0"))
```
Why change only the method call creation and not the search?   Could you also add a test for this?

Reply:
>  Could you also add a test for this?

This fixes roottest-cling-function-runlongArgTest - i.e. we have a test that is currently marked `WILLFAIL` and gets fixed by this PR.

Reply:
Well, I was expecting that this test now passes, and this ctest-fails (because marked `WILLFAIL`). Alas, the ref file is completely bogus, I don't know how / whether this ever worked, even under its old name `cint/function/longArgTest.ref`. Aaanyway: https://github.com/root-project/root/pull/12956 takes care of all of this.